semgrep-core/src/analyzing/AST_to_IL.ml

```diff
@@ -928,6 +973,69 @@ let rec stmt_aux env st =
   | G.OtherStmtWithStmt _ ->
       todo (G.S st)
 
+(* TODO: Maybe this and the following function could be merged *)
+and scrut_and_cases_to_exp env tok scrut_orig scrut cases =
+  (* If there is a scrutinee, the cases are expressions we need to check for equality with the scrutinee  *)
```

Comment:
ah scrutinee. Is it a standard name? I'd rather use expr, or switch_expr

Reply:
yeah it's definitely a name I've seen before. I can change it